### PR TITLE
File name on upload page

### PR DESCRIPTION
**Pull request by tsop14**
File name appears below the thumbnail. Ellipsis text overflow if long file name. 

screenshot-media local dev-gutools co uk 2015-09-03 17-14-45

screenshot-media local dev-gutools co uk 2015-09-03 17-14-30

### CODE REVIEW

**Comment by akash1810** [comment]
Any reason why you chose to put it under the thumbnail vs. a field on the right? As the filename in the first image is a random string, it looks a bit strange/random at first...

**Comment by theefer** [comment]
I don't have strong opinions on where it goes, the only thing I'll point out is that unlike the rest of the forms on the right, filename is not editable.

File: kahuna/public/js/edits/image-editor.html [hunk 1]
```diff
@@ -73,8 +73,11 @@
                 class="result-editor__archiver"
                 image="ctrl.image"
             ></ui-archiver>
-        </div>
 
+            <div class="result-editor__meta-data">
+                <span>{{ctrl.image.data.uploadInfo.filename}}</span>
```
Comment (akash1810) [inline kahuna/public/js/edits/image-editor.html]: can we add a `title` attribute for the tooltip?

**Comment by alastairjardine** [comment]
Pure opinion, but under the thumbnail's a bit odd. Would be better inlined on the right as a read only field. Consider it a title, I guess.

**Comment by paperboyo** [comment]
So does it mean that filename doesn’t go into a Desc field (when that’s empty) any more?

+1 for tooltip (in general tooltips should display the full thing whenever the thing is truncated for all thingz)



> the only thing I'll point out is that unlike the rest of the forms on the right, filename is not editable.

+1 Purer opinion than the rest: under thumbnail :stuck_out_tongue_winking_eye:

**Comment by theefer** [comment]
I think it should still also go in description if description is empty as apparently sometimes that's a useful bootstrap for the description.

**Comment by paperboyo** [comment]
It currently does exactly that. I was just asking if it’s gonna change seeing the first screenshot (I don’t think it should, as it’s useful proxy).

**Comment by tsop14** [comment]
screenshot-media local dev-gutools co uk 2015-09-03 18-02-58

**Comment by tsop14** [comment]
screenshot-media local dev-gutools co uk 2015-09-03 18-42-00

**Comment by theefer** [comment]
We may be able to find a visual treatment to differentiate it more from other writeable fields (esp Usage Rights, which looks identical but *is* editable).

As a next step/PR I suggest moving Usage rights to the top of the fields, as per @tsop14's second screenshot. Might mitigate the confusion.

**Comment by theefer** [comment]
Ah sorry missed out that Usage rights is already moved. Fine by me!

:+1:

**Comment by jamesgorrie** [comment]
:+1:

**Comment by akash1810** [comment]
> esp Usage Rights, which looks identical but is editable

That didn't even occur to me :flushed:.

:+1:

**Pull request merged**